client/app/resources/i18n/mxPlan/Messages_fr_FR.xml

```diff
@@ -5,6 +5,8 @@
    <translation id="mxPlan_order_step1_title" qtlid="260026">1. Configuration du MX Plan</translation>
    <translation id="mxPlan_order_step1_domain" qtlid="8458">Domaine</translation>
    <translation id="mxPlan_order_step1_offer" qtlid="50254">Type d'offre</translation>
+   <translation id="mxPlan_order_step1_expiration_warning">Veuillez noter que ce produit sera définitivement arrêté en décembre 2018. Sa commande reste cependant disponible pour une durée limitée. </translation>
+   <translation id="mxPlan_order_step1_expiration_warning_link">Cliquez ici pour plus d’informations</translation>
```

Comment:
Ce n'est pas super explicite comme label :)

Il sert à quoi ce bouton ? :)

Reply:
It's used to have more information about why the service will be stopped. It's pretty clear when you see the result.